flict/flictlib/project/writer.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-FileCopyrightText: 2024 Henrik Sandklef
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+def create_dummy_project(inbound, outbound):
```

Comment:
Do we really need to ship that as part of the library? Looks like test code to me

Reply:
removed